core/src/main/java/com/datastax/oss/driver/api/core/metrics/CoreNodeMetric.java

```diff
@@ -23,12 +23,12 @@
   POOLED_CONNECTION_COUNT("pooled_connection_count"),
   AVAILABLE_STREAM_COUNT("available_stream_count"),
   CQL_MESSAGES("cql_messages"),
-  WRITE_ERRORS("errors.write_errors"),
-  ABORTED_REQUESTS("errors.aborted_requests"),
-  WRITE_TIMEOUTS("errors.write_timeouts"),
-  READ_TIMEOUTS("errors.read_timeouts"),
-  UNAVAILABLES("errors.unavailables"),
-  OTHER_ERRORS("errors.others"),
+  UNSENT_REQUESTS("errors.request.unsent"),
```

Comment:
Renamed this because "write errors" was too similar to "write timeouts".

Reply:
👍 i also like what you did with the hierarchy (`errors.request` / `errors.connection`)